src/sentry/static/sentry/app/components/organizations/globalSelectionHeader/index.jsx

```diff
@@ -143,7 +141,7 @@ class GlobalSelectionHeader extends React.Component {
       // update URL parameters to reflect current store
       const {datetime, environments, projects} = selection;
 
-      if (hasMultipleProjectFeature || projects.length === 1) {
+      if (this.hasMultipleProjectSelection || projects.length === 1) {
```

Comment:
I know it's trivial but calling the function twice doesn't really seem like an improvement to me